tests/async.rs

```diff
@@ -102,10 +102,14 @@ where
             MysqlIntermediary::run_on_tcp(self, s)
         });
 
-        tokio::runtime::current_thread::block_on_all(
-            mysql_async::Conn::new(format!("mysql://127.0.0.1:{}", port)).and_then(|conn| c(conn)),
-        )
-        .unwrap();
+        // Create the runtime
+        let rt = tokio::runtime::Runtime::new().unwrap();
+
+        rt.spawn_blocking(move || {
```

Comment:
Why does this need `spawn_blocking`?

Reply:
I will remove the dependency on tokio. I checked it and we don't need tokio.

Reply:
Oh, that's weird. I wasn't actually proposing you remove tokio. Instead, I was just questioning the use of `spawn_blocking` specifically. I would think that `rt.block_on` would be sufficient if you remove the call to `.wait()`. The `.wait()` is just a utility function inside the mysql library that does that blocking for you. But using `.wait()` is also okay if you prefer that, it's just a bit less idiomatic.

Reply:
Sorry, I will fix it later :( I was not thinking that much.

Reply:
I guess we'll have to rely on `wait()` for now. Is it okay to change it to this?

```rust
        let rt = tokio::runtime::Runtime::new().unwrap();

        rt.block_on(async move {
            mysql_async::Conn::new(format!("mysql://127.0.0.1:{}", port))
                .and_then(|conn| c(conn))
                .wait()
                .unwrap()
        });
```

Or would you like to commit a patch telling me what to do? cc @jonhoo

Reply:
I was thinking:
```rust
rt.block_on(async move {
    mysql_async::Conn::new(format!("mysql://127.0.0.1:{}", port))
        .and_then(|conn| c(conn))
        .await.unwrap()
});
```
The `wait` method is just a helper method that `mysql_async` provides, but since we actually execute in async context here, we can do it properly and use `await`.